ladybug/sunpath.py

```diff
@@ -683,3 +756,21 @@ def __repr__(self):
             self.sunVector.y,
             self.sunVector.z
         )
+
+
+# Following are test cases to compare altitude and azimuth values with NOAA calculator
```

Comment:
Thanks for adding these cases. They either need to be written as a proper test cases or at least we should make sure that they won't be execute on import. To achieve that you can use `if __name__ == '__main__:`. If you don't know the concept watch [this video](https://www.youtube.com/watch?v=sugvnHA7ElY).

Once you add the if statement I can merge this. I will change them to proper test cases afterwards and you can follow that as an example for later.

Thanks again for all the great work @devngc!